### PR TITLE
Reorganize examples and provide fraud detection example

### LINKED ISSUE #147: Move example notebooks to separate repo
To keep this repo a bit tidier and more consistent with other RAPIDS projects, we should move the current example notebook to the RAPIDS examples repo. This should happen over two releases, with a deprecation notice added in the first release to allow users to update links and bookmarks.

### PR DESCRIPTION
**Pull request by wphicks**
Reorganize notebooks to allow for easier addition of further examples. Add fraud detection example with categorical features.

Resolve #147

### CODE REVIEW

File: notebooks/categorical-fraud-detection/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Fraud Detection With Categorical XGBoost
+
+This example notebook demonstrates how to train and deploy an XGBoost model
```
Comment (dantegd) [inline notebooks/categorical-fraud-detection/README.md:31]:
```suggestion
This example notebook shows how to train and deploy an XGBoost model
```

File: notebooks/categorical-fraud-detection/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```bash
+conda activate triton_example
+jupyter notebook
+```
+The Jupyter interface should now be accessible from a browser, and you can
+follow the instructions from there.
+
+**NOTE**: You will also need [Docker](https://docs.docker.com/get-docker/)
+available on your system in order to run the Triton server as part of the
+example notebook.
```
Comment (dantegd) [inline notebooks/categorical-fraud-detection/README.md:59]: I think the note should be towards the beginning of the section

File: notebooks/simple-xgboost/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Trtion FIL backend with XGBoost
+
+This notebook is a reference for deploying a forest model trained using XGBoost library in Triton Inference Server with Forest Inference Library (FIL) backend. The notebook explains how one can deploy XGBoost model in Triton, check deployment status and send inference requests, set concurrent model execution and dynamic batching and find the best deployment configuration using Model Analyzer.
```
Comment (dantegd) [inline notebooks/simple-xgboost/README.md:3]: Seems a bit odd to have the acronyms expanded only here and nowhere else

File: notebooks/simple-xgboost/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Requirements
+* Nvidia GPU (Pascal+ Recommended GPUs: T4, V100 or A100)
```
Comment (dantegd) [inline notebooks/simple-xgboost/README.md:6]:
```suggestion
* NVIDIA GPU (Pascal required, recommended GPUs: T4, V100 or A100)
```

**Review by dantegd — APPROVED** [review]
Pre-approving since readme's look great, just had a few suggestions

File: notebooks/simple-xgboost/README.md (hunk 3 above)
Comment (wphicks) [reply notebooks/simple-xgboost/README.md:3]: Agreed. I'm trying not to make too significant of changes to the stuff that was here already, but that seems worth a touchup. I'll take care of it.

**Pull request merged**